spanner/README.rst

```diff
@@ -12,3 +12,132 @@ Quick Start
 .. code-block:: console
 
     $ pip install --upgrade google-cloud-spanner
+
+
+Authentication
+--------------
+
+With ``google-cloud-python`` we try to make authentication as painless as
+possible. Check out the `Authentication section`_ in our documentation to
+learn more. You may also find the `authentication document`_ shared by all
+the ``google-cloud-*`` libraries to be helpful.
+
+.. _Authentication section: https://google-cloud-python.readthedocs.io/en/latest/core/auth.html
+.. _authentication document: https://github.com/GoogleCloudPlatform/gcloud-common/tree/master/authentication
+
+
+Using the API
+-------------
+
+Cloud Spanner is the world’s first fully managed relational database service
+to offer both strong consistency and horizontal scalability for
+mission-critical online transaction processing (OLTP) applications. With Cloud
+Spanner you enjoy all the traditional benefits of a relational database; but
+unlike any other relational database service, Cloud Spanner scales
+horizontally to hundreds or thousands of servers to handle the biggest
+transactional workloads. (`About Cloud Spanner`_)
+
+.. _About Cloud Spanner: https://cloud.google.com/spanner/
+
+
+Executing Arbitrary SQL in a Transaction
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Generally, to work with Cloud Spanner, you will want a transaction. The
+preferred mechanism for this is to create a single function, which executes
+as a callback to ``database.run_in_transaction``:
+
+.. code:: python
+
+    # First, define the function that represents a single "unit of work"
+    # that should be run within the transaction.
+    def update_anniversary(transaction, person_id, unix_timestamp):
+        # The query itself is just a string.
+        #
+        # The use of @parameters is recommended rather than doing your
+        # own string interpolation; this provides protections against
+        # SQL injection attacks.
+        query = """UPDATE people
+            SET anniversary = @uxts
+            WHERE id = @person_id"""
```

Comment:
Sorry, I am confused. I pulled this (not this exact query, but the concept) from [the Spanner docs](https://cloud.google.com/spanner/docs/reference/rpc/google.spanner.v1#google.spanner.v1.ExecuteSqlRequest).

How _should_ this be written?